tsconfig.json

```diff
@@ -1,17 +1,12 @@
 {
   "compileOnSave": true,
   "compilerOptions": {
-    // "declaration": true,
-    "target": "es5",
-    "lib": [
-      "es5",
-      "es6",
-      "es2015",
-      "es2016",
-      "es2017"
-    ],
+    "declaration": true,
+    "target": "es2017",  // fully supported with node ^8.10.0 (cf. https://node.green/)
```

Comment:
I think only es5 will work with Apps Script.

Reply:
this is the config used on ts2gas.ts (and test.ts), hence the target is node (^8.10.0) not GAS

Reply:
Oh, thats fine.